### PR TITLE
[verifier] Upgrade to RN 61

### LINKED ISSUE #1574: [verifier] App doesn't build
### Expected Behavior

`yarn dev` should start up the app. Has not been updated to RN 61.

### Current Behavior

Build fails.

### PR DESCRIPTION
**Pull request by martinvol**
### Description

Verifier app hadn't been migrated to RN 61 and didn't build. So far it builds but yarn can't find 'react-native'

### Tested

- builds and updates tests

### Other changes

- 

### Related issues

- closes #1574

### Backwards compatibility

-

### CODE REVIEW

**Comment by martinvol** [comment]
Not sure if I should change 
https://github.com/celo-org/celo-monorepo/blob/e2f6485e8c505bce37aabdab0f4dc3379f99dbbc/packages/verifier/android/app/src/main/AndroidManifest.xml#L19

for 

```
<service android:name="io.invertase.firebase.messaging.RNFirebaseMessagingService">
      <intent-filter>
        <action android:name="com.google.firebase.MESSAGING_EVENT" />
      </intent-filter>
    </service>
```
Like in the app's manifest.

**Comment by martinvol** [comment]
The app now builds but the `metro bundler` fails as soon as the build request the bundle:

```
$ yarn start --reset-cache
yarn run v1.17.3
$ react-native start --reset-cache
warn Your project is using deprecated "rnpm" config that will stop working from next release. Please use a "react-native.config.js" file to configure the React Native CLI. Migration guide: https://github.com/react-native-community/cli/blob/master/docs/configuration.md
┌──────────────────────────────────────────────────────────────────────────────┐
│                                                                              │
│  Running Metro Bundler on port 8081.                                         │
│                                                                              │
│  Keep Metro running while developing on any JS projects. Feel free to        │
│  close this tab and run your own Metro instance if you prefer.               │
│                                                                              │
│  https://github.com/facebook/react-native                                    │
│                                                                              │
└──────────────────────────────────────────────────────────────────────────────┘

Looking for JS files in
   /Users/martinvol/celo/celo-monorepo/packages/verifier
   /Users/martinvol/celo/celo-monorepo

warning: the transform cache was reset.
Loading dependency graph, done.
error: bundling failed: Error: Unable to resolve module `react-native` from `index.js`: react-native could not be found within the project or in these directories:
  ../../node_modules
  /Users/martinvol/celo/celo-monorepo/packages/verifier/node_modules

If you are sure the module exists, try these steps:
 1. Clear watchman watches: watchman watch-del-all
 2. Delete node_modules: rm -rf node_modules and run yarn install
 3. Reset Metro's cache: yarn start --reset-cache
 4. Remove the cache: rm -rf /tmp/metro-*
    at ModuleResolver.resolveDependency (/Users/martinvol/celo/celo-monorepo/node_modules/metro/src/node-haste/DependencyGraph/ModuleResolution.js:186:15)
    at ResolutionRequest.resolveDependency (/Users/martinvol/celo/celo-monorepo/node_modules/metro/src/node-haste/DependencyGraph/ResolutionRequest.js:52:18)
    at DependencyGraph.resolveDependency (/Users/martinvol/celo/celo-monorepo/node_modules/metro/src/node-haste/DependencyGraph.js:282:16)
    at Object.resolve (/Users/martinvol/celo/celo-monorepo/node_modules/metro/src/lib/transformHelpers.js:267:42)
    at dependencies.map.result (/Users/martinvol/celo/celo-monorepo/node_modules/metro/src/DeltaBundler/traverseDependencies.js:426:31)
    at Array.map (<anonymous>)
    at resolveDependencies (/Users/martinvol/celo/celo-monorepo/node_modules/metro/src/DeltaBundler/traverseDependencies.js:423:18)
    at /Users/martinvol/celo/celo-monorepo/node_modules/metro/src/DeltaBundler/traverseDependencies.js:275:33
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/Users/martinvol/celo/celo-monorepo/node_modules/metro/src/DeltaBundler/traverseDependencies.js:87:24)
 BUNDLE  [android, dev] ./index.js ░░░░░░░░░░░░░░░░ 0.0% (0/1)::ffff:127.0.0.1 - - [04/Nov/2019:14:12:46 +0000] "GET /index.bundle?platform=android&dev=true&minify=false HTTP/1.1" 500 - "-" "okhttp/3.12.1"
 BUNDLE  [android, dev] ./index.js ░░░░░░░░░░░░░░░░ 0.0% (0/1), failed.
```

File: packages/verifier/src/components/HomeScreen/HomeScreen.tsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ class HomeScreen extends React.Component<Props, State> {
 
   async componentDidMount() {
     try {
-      NetInfo.addEventListener('connectionChange', this.handleNetworkStatusChange)
+      NetInfo.addEventListener(this.handleNetworkStatusChange)
```
Comment (martinvol) [inline packages/verifier/src/components/HomeScreen/HomeScreen.tsx:90]: API changed and I was getting a warning

**Review by jmrossy — APPROVED** [review]
Nice!!

**Pull request merged**